src/com/facebook/buck/swift/SwiftLibraryDescription.java

```diff
@@ -207,6 +210,24 @@ public BuildRule apply(SwiftLibrary input) {
                 }
               })
               .toSortedSet(Ordering.natural()));
+
+      if (args.exportedHeaders.isPresent()) {
+        UnflavoredBuildTarget unflavoredBuildTarget =
+            params.getBuildTarget().getUnflavoredBuildTarget();
+
+        for (HeaderVisibility headerVisibility : HeaderVisibility.values()) {
+          // unflavoredBuildTarget because #headers can collide with any other flavor
+          // from the same domain.
+          CxxDescriptionEnhancer.requireHeaderSymlinkTree(
```

Comment:
It looks like the flavor isn't actually used for headers; you should be able to safely remove it from `AppleTestDescription` and `AppleTestIntegrationTest`.

cc: @k21 who introduced it just in case there's something I overlooked.